app/controllers/bulk_downloads_controller.rb

```diff
@@ -54,6 +54,7 @@ def create
       rescue => e
         # If the kickoff failed, set to error.
         bulk_download.update(status: BulkDownload::STATUS_ERROR)
+        Rails.logger.error(e)
```

Comment:
`log_err_and_airbrake` already does this. you do not need it. 
On the other hand we might not have the full stack trace.